doc/yjit/yjit.md

```diff
@@ -165,9 +165,11 @@ YJIT supports all command-line options supported by upstream CRuby, but also add
 
 - `--yjit`: enable YJIT (disabled by default)
 - `--yjit-call-threshold=N`: number of calls after which YJIT begins to compile a function (default 30)
+- `--yjit-cold-threshold=N`: number of calls after which an ISEQ is considered cold and not
```

Comment:
```suggestion
- `--yjit-cold-threshold=N`: number of global calls after which an ISEQ is considered cold and not
```
thinking it might be a bit puzlli gotherwise as it;s right after the call threshold that uses `calls` as "calls to that one function (we should rename to method probably). While `cold-threshold` is the global number of method calls.